src/DotNet.Status.Web/ArcadeValidationData.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DotNet.Status.Web
+{
+    public class ArcadeValidationData
+    {
+        public DateTime BuildDateTime { get; set; }
```

Comment:
You probably want to slap "[Required]" on all these properties, since I don't think the method will do great things if someone leaves one off. :-) It will make the swagger mark them as required, and then, consequently, any swagger generated clients.